[Docs] Rewrite explore section of docs using yugabyted (#8678)
yugabyte/yugabyte-db: docs/content/latest/explore/linear-scalability/scaling-transactions.md

-->
</ul>

> We’ll begin our discussion by seeing horizontal scale-out and scale-in in action. In particular, we’ll see how in YugabyteDB, you can add nodes to scale your cluster up very efficiently and reliably in order to achieve more read and write IOPS (input/output operations per second). In this tutorial, you will look at how YugabyteDB can scale while a workload is running. You will run a read-write workload using the prepackaged [YugabyteDB workload generator](https://github.com/yugabyte/yb-sample-apps) against a 3-node local cluster with a replication factor of 3, and add nodes to it while the workload is running. Next, you can observe how the cluster scales out by verifying that the number of read and write IOPS are evenly distributed across all the nodes at all times.

Reviewer: Don't use first-person.

```suggestion
On this page, you'll observe horizontal scale-out and scale-in in action. In particular, you’ll see how in YugabyteDB, you can add nodes to scale your cluster up very efficiently and reliably to achieve more read and write IOPS (input/output operations per second), even while a workload is running. You'll run a read-write workload using the prepackaged [YugabyteDB workload generator](https://github.com/yugabyte/yb-sample-apps) against a 3-node local cluster with a replication factor of 3, and add nodes to it while the workload is running. Next, you can observe how the cluster scales out by verifying that the number of read and write IOPS are evenly distributed across all the nodes at all times.
```

This tutorial uses the [yugabyted](../../../reference/configuration/yugabyted) cluster management utility.

## 1. Create universe

> Start a new three-node cluster with a replication factor (RFå) of `3` and set the number of [shards](../../../architecture/concepts/docdb/sharding/) (aka tablets) per table per YB-TServer to `4` so that you can better observe the load balancing during scale-up and scale-down. <br />

Reviewer: Use this link everywhere (the old one is a redirect, it turns out)

```suggestion
Start a new three-node cluster with a replication factor (RF) of `3` and set the number of [shards](../../../architecture/docdb-sharding/sharding/) (also called tablets) per table per YB-TServer to `4` so that you can better observe the load balancing during scale-up and scale-down. <br />
```

                  --tserver_flags "ysql_num_shards_per_tserver=4,follower_unavailable_considered_failed_sec=30"
```

Now you should have 4 nodes. Refresh the <a href='http://127.0.0.1:7000/tablet-servers' target="_blank">tablet-servers</a> page to see the statistics update. Shortly, you should see the new node performing a comparable number of reads and writes as the other nodes. The 36 tablets will now get distributed evenly across all the 4 nodes, leading to each node having 9 tablets.

The YugabyteDB universe automatically lets the client know to use the newly added node for serving queries. This scaling out of client queries is completely transparent to the application logic, allowing the application to scale linearly for both reads and writes.

![Read and write IOPS with 4 nodes - Rebalancing in progress](/images/ce/transactions_newnode_adding_observe.png)

![Read and write IOPS with 4 nodes](/images/ce/transactions_newnode_added_observe.png)

## 5. Remove node and observe linear scale in

Remove the recently added node from the universe.

```sh
$ ./bin/yugabyted stop \
                  --base_dir=/tmp/ybd4
```

> Refresh the <a href='http://127.0.0.1:7000/tablet-servers' target="_blank">tablet-servers</a> page to see the stats update. The `Time since heartbeat` value for that node will keep increasing. Once that number reaches 60s (i.e. 1 minute), YugabyteDB will change the status of that node from ALIVE to DEAD. We can now see the load (tablets) and IOPS getting moved off the removed node and redistributed amongst the other nodes.

Reviewer: ```suggestion
Refresh the <a href='http://127.0.0.1:7000/tablet-servers' target="_blank">tablet-servers</a> page to see the stats update. The `Time since heartbeat` value for that node will keep increasing. Once that number reaches 60s (i.e. 1 minute), YugabyteDB will change the status of that node from ALIVE to DEAD. Observe the load (tablets) and IOPS getting moved off the removed node and redistributed amongst the other nodes.
```

![Read and write IOPS with 4th node dead](/images/ce/transactions_deleting_observe.png)

![Read and write IOPS with 4th node removed](/images/ce/transactions_deleted_observe.png)

## 6. Clean up (optional)

> Optionally, you can shutdown the local cluster created in Step 1.

Reviewer: ```suggestion
Optionally, you can shut down the local cluster created in Step 1.
```
